server/views/grid/aligning/README.md

```diff
@@ -1 +1,3 @@
 Applying a class of either `grid--start`, `grid--center` or `grid--end` to the containing `grid` div will determine how grid cells are aligned horizontally.
+
+Applying a class of `grid--spaced` to the containing `grid` element will space out its child `grid__cell`s (useful for when e.g. there is an empty column of space between to content columns, rather than having to include an empty element for spacing).
```

Comment:
Cool - good to know.

We are going to probably start looking at creating guidelines at some stage - it'd be good to have to have a think about it - do we break on test if someone added more than two - how could we test for this etc.

An example would be copy styleguide - instead of having an editor sub-edit another person's content saying, "Of course we use the oxford comma" - you could have the tool suggest that in the first place...

Worth a think.